packages/directory/tests/dummy/app/router.js

```diff
@@ -8,6 +9,17 @@ const Router = EmberRouter.extend({
 
 Router.map(function() {
   this.route('directory', {path: '/'});
+
+  this.route('reports', function() {
+    this.route('new');
+    this.route('report', { path: '/:reportId'}, function() {
+      this.route('new');
```

Comment:
Do we need both the new routes? line 14 and 16.

Reply:
Yes, `reports.new` generates the temp id for the new report path.